services/QuillLMS/client/app/bundles/PremiumHub/components/CanvasInstanceSchoolsSection.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+const smallWhiteCheckSrc = `${process.env.CDN_URL}/images/shared/check-small-white.svg`
+
+const CanvasInstanceSchoolRow = ({ school, setSelectedSchoolIds, selectedSchoolIds, }) => {
+  function handleClickCheckbox() {
+    if (selectedSchoolIds.includes(school.id)) {
+      setSelectedSchoolIds(selectedSchoolIds.filter(id => id !== school.id))
+    } else {
+      setSelectedSchoolIds(selectedSchoolIds.concat(school.id))
+    }
+  }
+
+  const checkbox = selectedSchoolIds.includes(school.id) ? (
+    <button
+      aria-label="Checked checkbox"
+      className="quill-checkbox selected"
+      onClick={handleClickCheckbox}
+      type="button"
+    >
+      <img alt="Checked checkbox" src={smallWhiteCheckSrc} />
+    </button>
+  ) : (
+    <button
+      aria-label="Unchecked checkbox"
+      className="quill-checkbox unselected"
+      onClick={handleClickCheckbox}
+      type="button"
+    />
+  )
```

Comment:
Do we not have a re-usable Checkbox component that handles this behavior for us?  That feels like something we should probably have standardized.  Probably out of scope for this PR, though, so you can probably leave this as-is.

Reply:
Yeah, we don't, we just use the reusable CSS for it right now. I'll make a card to get to that at some point.